### ISSUE TITLE
Unable to install plugin 

### ISSUE THREAD

**Issue by ThisDrunkDane** [lbrlabs/pulumi-scaleway#130]
From the documentation it's stated you simply should be able to simply type `pulumi plugin install resource scaleway <version>` and have the provider installed, but it seems to 404 when I try, do we have to use some `--server` to install the plugin or has the latest release not been propagated to the registry?

```bash
❯ pulumi plugin install resource scaleway v1.0.0
[resource plugin scaleway-1.0.0] installing
warning: Error downloading plugin: 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-scaleway-v1.0.0-linux-amd64.tar.gz
Will retry in 80ms [1/5]
warning: Error downloading plugin: 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-scaleway-v1.0.0-linux-amd64.tar.gz
Will retry in 160ms [2/5]
warning: Error downloading plugin: 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-scaleway-v1.0.0-linux-amd64.tar.gz
Will retry in 320ms [3/5]```

**Comment by ThisDrunkDane** [comment]
I guess a workaround for now is `--server https://github.com/lbrlabs/pulumi-scaleway/releases/download/v1.0.0` but that doesn't seem intended?

**Comment by jaxxstorm** [comment]
@ThisDrunkDane are you using the TypeScript SDK? There's currently a bug with the way plugin acquisition works for third-party resources that are in the Pulumi SDK.

**Comment by ThisDrunkDane** [comment]
No I'm using the Go SDK, but even without all that I'm just calling it using the CLI for now

**Comment by jaxxstorm** [comment]
This should be resolved as of v1.1.0, but please reopen if it's not

**Comment by AzSiAz** [comment]
As I am new to pulumi I am not sure if it's resolved as running:

```bash
pulumi plugin install resource scaleway v1.2.0
```

Still result in error:
```
error: [resource plugin scaleway-1.2.0] downloading from : failed to download plugin: scaleway-1.2.0: 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-scaleway-v1.2.0-linux-amd64.tar.gz 
```
